twitter_ads/resource.py

```diff
@@ -133,6 +134,50 @@ def _load_resource(self, klass, id, **kwargs):
             return klass.load(self, id, **kwargs)
 
 
+class Batch(object):
+
+    _ENTITY_MAP = {
+        'LineItem': ENTITY.LINE_ITEM,
+        'Campaign': ENTITY.CAMPAIGN,
+        'TargetingCriteria': ENTITY.TARGETING_CRITERION
+    }
+
+    @classmethod
+    def batch_save(klass, account, objs):
+        """
+        Makes batch request(s) for a passed in list of objects
+        """
+
+        resource = klass.BATCH_RESOURCE_COLLECTION.format(account_id=account.id)
+
+        json_body = []
+
+        for obj in objs:
+            entity_type = klass._ENTITY_MAP[klass.__name__].lower()
+            obj_json = {'params': obj.to_params()}
+
+            if obj.id is None:
```

Comment:
Based upon how Python is so adept at handling dictionaries/json handling was trying to think of a way we could do this without the if/else blocks or logic here (risk for bugs)